.circleci/config.yml

```diff
@@ -28,8 +28,7 @@ jobs:
       - run: make build
       - run: make test-unit
       - run: make test-integration-cover
-      # fixme: removed this while coveralls is down. Re-add in future
```

Comment:
It is a comment for we know why the line is commented. We need to come back it gain when the coveralls be fixed.